### ISSUE TITLE
Possible bug in implementation of PPO Clipped Surrogate Objective

### ISSUE THREAD

**Issue by aishwaryap** [openai/baselines#1156]
I believe there might be a bug in the implementation of the Clipped Surrogate Objective in PPO [here](https://github.com/openai/baselines/blob/master/baselines/ppo2/model.py#L86). 
According to Equation 7 in [the PPO paper](https://arxiv.org/pdf/1707.06347.pdf), I would expect that line to be
```
pg_loss = tf.reduce_mean(tf.minimum(pg_losses, pg_losses2))
```

In case this is not a bug, I would appreciate it if someone could explain to me why it's maximum in the code?

**Comment by joschu** [comment]
Hi Aishwarya, the paper used an objective that should be maximized, whereas the code uses a loss that should be minimized. You can verify that if you flip it the wrong way, the algorithm doesn't work at all.
